+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Sat Aug 26 17:01:48 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL iputils
+ pkgs=iputils
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs iputils --node 1 --tempfile ''
Sat Aug 26 17:01:48 2023 - INFO  - pkgs:(iputils) is already installed
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Sat Aug 26 17:01:48 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sat Aug 26 17:01:48 2023 - INFO  - Start to run test.
+ tracepath6 -V
+ grep 'tracepath from iputils'
tracepath from iputils 20221126
+ CHECK_RESULT 0 0 0 'tracepath6 -V execute failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='tracepath6 -V execute failed'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ tracepath6 -4 ''
+ grep tracepath6
tracepath6: Only one -4 or -6 option may be specified
+ CHECK_RESULT 1 1 0 'tracepath6 -4 execute failed'
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log='tracepath6 -4 execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ tracepath6 -6 ''
+ grep pmtu
tracepath6: : Name or service not known
+ CHECK_RESULT 1 0 0 'tracepath6 -6 execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='tracepath6 -6 execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'tracepath6 -6 execute failed'
+ LOG_ERROR 'tracepath6 -6 execute failed'
+ message='tracepath6 -6 execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'tracepath6 -6 execute failed'
Sat Aug 26 17:01:48 2023 - ERROR - tracepath6 -6 execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_tracepath6.sh line 35'
+ message='oe_test_iputils_tracepath6.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_tracepath6.sh line 35'
Sat Aug 26 17:01:48 2023 - ERROR - oe_test_iputils_tracepath6.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ tracepath6 -b ''
+ grep pmtu
tracepath6: : Name or service not known
+ CHECK_RESULT 1 0 0 'tracepath6 -b execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='tracepath6 -b execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'tracepath6 -b execute failed'
+ LOG_ERROR 'tracepath6 -b execute failed'
+ message='tracepath6 -b execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'tracepath6 -b execute failed'
Sat Aug 26 17:01:48 2023 - ERROR - tracepath6 -b execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_tracepath6.sh line 37'
+ message='oe_test_iputils_tracepath6.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_tracepath6.sh line 37'
Sat Aug 26 17:01:48 2023 - ERROR - oe_test_iputils_tracepath6.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ tracepath6 -l 65534 ''
+ grep pmtu
tracepath6: : Name or service not known
+ CHECK_RESULT 1 0 0 'tracepath6 -l execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='tracepath6 -l execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'tracepath6 -l execute failed'
+ LOG_ERROR 'tracepath6 -l execute failed'
+ message='tracepath6 -l execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'tracepath6 -l execute failed'
Sat Aug 26 17:01:49 2023 - ERROR - tracepath6 -l execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_tracepath6.sh line 39'
+ message='oe_test_iputils_tracepath6.sh line 39'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_tracepath6.sh line 39'
Sat Aug 26 17:01:49 2023 - ERROR - oe_test_iputils_tracepath6.sh line 39
+ '[' 0 -eq 1 ']'
+ return 0
+ grep pmtu
+ tracepath6 -m 10 ''
tracepath6: : Name or service not known
+ CHECK_RESULT 1 0 0 'tracepath6 -m execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='tracepath6 -m execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'tracepath6 -m execute failed'
+ LOG_ERROR 'tracepath6 -m execute failed'
+ message='tracepath6 -m execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'tracepath6 -m execute failed'
Sat Aug 26 17:01:49 2023 - ERROR - tracepath6 -m execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_tracepath6.sh line 41'
+ message='oe_test_iputils_tracepath6.sh line 41'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_tracepath6.sh line 41'
Sat Aug 26 17:01:49 2023 - ERROR - oe_test_iputils_tracepath6.sh line 41
+ '[' 0 -eq 1 ']'
+ return 0
+ tracepath6 -n ''
+ grep pmtu
tracepath6: : Name or service not known
+ CHECK_RESULT 1 0 0 'tracepath6 -n execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='tracepath6 -n execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'tracepath6 -n execute failed'
+ LOG_ERROR 'tracepath6 -n execute failed'
+ message='tracepath6 -n execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'tracepath6 -n execute failed'
Sat Aug 26 17:01:49 2023 - ERROR - tracepath6 -n execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_tracepath6.sh line 43'
+ message='oe_test_iputils_tracepath6.sh line 43'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_tracepath6.sh line 43'
Sat Aug 26 17:01:49 2023 - ERROR - oe_test_iputils_tracepath6.sh line 43
+ '[' 0 -eq 1 ']'
+ return 0
+ grep pmtu
+ tracepath6 -p 15 ''
tracepath6: : Name or service not known
+ CHECK_RESULT 1 0 0 'tracepath6 -p execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='tracepath6 -p execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'tracepath6 -p execute failed'
+ LOG_ERROR 'tracepath6 -p execute failed'
+ message='tracepath6 -p execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'tracepath6 -p execute failed'
Sat Aug 26 17:01:49 2023 - ERROR - tracepath6 -p execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_tracepath6.sh line 45'
+ message='oe_test_iputils_tracepath6.sh line 45'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_tracepath6.sh line 45'
Sat Aug 26 17:01:49 2023 - ERROR - oe_test_iputils_tracepath6.sh line 45
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Sat Aug 26 17:01:49 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sat Aug 26 17:01:49 2023 - ERROR - failed to execute the case.
+ exit 6
+ post_test
+ LOG_INFO 'start to restore the test environment.'
+ message='start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start to restore the test environment.'
Sat Aug 26 17:01:49 2023 - INFO  - start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Sat Aug 26 17:01:49 2023 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Sat Aug 26 17:01:49 2023 - INFO  - End to restore the test environment.
